### PR TITLE
Update how-to-change-fonts-and-colors-in-the-editor.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
This was hard to find from google. VSC takes all the top spots. Hoping to boost SEO

### CODE REVIEW

**Comment by PRMerger10** [comment]
@Rick-Anderson : Thanks for your contribution! The author, @gewarren, has been notified to review your proposed change.

File: docs/ide/reference/how-to-change-fonts-and-colors-in-the-editor.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Change editor fonts and colors
+title: Change editor fonts and colors in Visual Studio 2017
```
Comment (gewarren) [inline docs/ide/reference/how-to-change-fonts-and-colors-in-the-editor.md]: "Visual Studio" is added as a title suffix globally, so we don't want to add it here. It ends up being duplicated in the title.

**Comment by Rick-Anderson** [comment]
Arg, I should have added the search. Now your doc is coming up first.

**Comment by PRMerger19** [comment]
@Rick-Anderson : Thanks for your contribution! The author, @gewarren, has been notified to review your proposed change.

**Pull request merged**